src/layout/Group/DisplayGroupContainer.tsx

```diff
@@ -9,7 +9,7 @@ import classes from 'src/layout/Group/DisplayGroupContainer.module.css';
 import { pageBreakStyles, selectComponentTexts } from 'src/utils/formComponentUtils';
 import { LayoutNode } from 'src/utils/layout/LayoutNode';
 import { getTextFromAppOrDefault } from 'src/utils/textResource';
-import type { HGroups } from 'src/utils/layout/hierarchy.types';
```

Comment:
<3

Reply:
I think this broke in main, I have had to do it in another PR as well or tests wont run 😅

Reply:
Same here..!